src/jetstream/plugins/epinio/main.go

```diff
@@ -36,30 +42,51 @@ func init() {
 // Init creates a new Analysis
 func Init(portalProxy interfaces.PortalProxy) (interfaces.StratosPlugin, error) {
 	if interfaces.AuthEndpointTypes[portalProxy.GetConfig().AuthEndpointType] != interfaces.Epinio {
-		return nil, fmt.Errorf("Epinio plugin requires auth endpoint type of %s", interfaces.Epinio)
+		return nil, fmt.Errorf("epinio plugin requires auth endpoint type of %s", interfaces.Epinio)
 	}
 
 	epinioApiUrlValue, _ := portalProxy.Env().Lookup(epinioApiUrlEnv)
 	if len(epinioApiUrlValue) == 0 {
-		return nil, fmt.Errorf("Failed to find Epinio API url env `%s`", epinioApiUrlEnv)
+		return nil, fmt.Errorf("failed to find Epinio API url env `%s`", epinioApiUrlEnv)
 	}
 
 	epinioApiWsUrlValue, _ := portalProxy.Env().Lookup(epinioApiWsUrl)
 	if len(epinioApiWsUrlValue) == 0 {
-		log.Warnf("Failed to find Epinio WS API url env `%s`", epinioApiWsUrl)
+		epinioApiWsUrlValue = strings.Replace(epinioApiUrlValue, "https://", "wss://", 1)
+		log.Infof("Didn't find `%s`, falling back to `%s`", epinioApiWsUrl, epinioApiWsUrlValue)
+
 	}
 
 	epinioApiUrlskipSSLValidation, err := portalProxy.Env().Bool(epinioApiUrlskipSSLValidationEnv)
 	if err != nil {
 		epinioApiUrlskipSSLValidation = false
 	}
 
-	log.Infof("Epinio API url: '%s'. Epinio WSS url: '%s'. Skipping SSL Validation: '%+v'", epinioApiUrlValue, epinioApiWsUrlValue, epinioApiUrlskipSSLValidation)
+	epinioAuthUrlValue, _ := portalProxy.Env().Lookup(epinioDexAuthUrl)
+	if len(epinioAuthUrlValue) == 0 {
+		epinioAuthUrlValue = strings.Replace(epinioApiUrlValue, "epinio.", "auth.", 1)
```

Comment:
I think this is wrong. IIUC it doesn't replace the internal `epinio-server` prefix but the namespace `epinio` value instead. So in the end the URL is wrong `http://epinio-server.auth.svc.cluster.local/`.
```suggestion
		epinioAuthUrlValue = strings.Replace(epinioApiUrlValue, "epinio-server.", "auth.", 1)
```

Reply:
This was an optimisation for the external scenario.

What's the correct internal url (as in, where does the `auth.<domain>` external url point to), something like...?
```
http://dex.epinio.svc.cluster.local
```

Reply:
yes `http://dex.epinio.svc.cluster.local` should be correct but it's not listening on port `80` but on `5556` and `5558`.

Ingress for `auth.<domain>` points to port `5556`.